mdr/mdr.py

```diff
@@ -73,7 +90,31 @@ def transform(self, features):
             Constructed features from the provided feature matrix
 
         """
-        return 0
+        new_feature = np.zeros(shape=(features.shape[0],1), dtype=np.int)
```

Comment:
current status: fixed all comments above, working on unit test, and will continue to commit after some more unit tests.